src/quokka_editor_back/routers/documents.py

```diff
@@ -59,9 +69,9 @@ async def create_document(
         content = document_template.content
 
     new_document = await Document.create(
-        title=title,
+        title="Draft Document",
```

Comment:
This looks like a merge conflict. Bring back assigning the title and content through variables.